Makefile

```diff
@@ -210,7 +211,10 @@ requirements_awx: virtualenv_awx
 requirements_awx_dev:
 	$(VENV_BASE)/awx/bin/pip install -r requirements/requirements_dev.txt
 
-requirements: requirements_ansible requirements_awx
+requirements_collections:
+	ansible-galaxy collection install -r requirements/collections_requirements.yml -p ${COLLECTION_BASE}
+
+requirements: requirements_ansible requirements_awx requirements_collections
```

Comment:
I don't know what exactly uses this target, like, what deployment methods share it.

`COLLECTION_BASE ?= /collections` seems like kind of a weird location for the folder. Myself, I would not hesitate to put this in the `/var/lib/awx/...` location

Reply:
I also don't see this invoked in the Dockerfile for the installer here or in the docker-compose setup.

This command can take a frustrating long period of time, and it certainly seems right to do it at the time of image building.

Reply:
@AlanCoding The PR submitted from last night was and still is a draft meant to show a working flow with package install. From an upstream perspective the changes to the various Dockerfiles have been added this morning.

Reply:
Also, I am still looking for guidance on where we would preferably have this laid down. Given `/var/lib/awx/collections` is not an option. Do we want to make it clear we are vendoring it by having it in `/var/lib/awx/vendor/collections` ?

Reply:
> Packaging.

but it's messier than that, right?

```
$ grep -r "make requirements" installer/
installer//roles/image_build/templates/Dockerfile.j2:RUN cd /tmp && VENV_BASE="/var/lib/awx/venv" make requirements_awx requirements_ansible_py3
```

https://github.com/ansible/awx/blob/eafd40291e786599c712b36bc4012199619a97fc/installer/roles/image_build/templates/Dockerfile.j2#L71

I think maybe we would add `make requirements_collections` there

Reply:
@AlanCoding its already there -> https://github.com/ansible/awx/pull/6264/files#diff-ea72c08e2b13ba6d1d9334f604684fe2R97-R98

Reply:
/var/lib/awx/vendor/inventory_collections ?